Tests/SonarScanner.MSBuild.Tasks.UnitTest/TaskUtilitiesTests.cs

```diff
@@ -30,6 +30,9 @@ namespace SonarScanner.MSBuild.Tasks.UnitTest
     [TestClass]
     public class TaskUtilitiesTests
     {
+        private const int MaxConfigRetryPeriodInMilliseconds = 2500;
```

Comment:
I'm not convinced about value of this change. It's more noisy and only thing it brings is that now the values CAN get out of sync easily. Before, they couldn't with no additional effort.